Makefile

```diff
@@ -1,19 +1,34 @@
-# The following parameters are required:
+# The following parameters are required by 'run':
 # - REPOSITORY
 # - CIRCLECI_API_KEY
 #
 
-AWS_ACCESS_KEY_ID := $(shell aws secretsmanager get-secret-value --secret-id=ci-user-aws-keys${randomSeed} | jq -r '.SecretString'| jq -r .access_key_id)
-AWS_SECRET_ACCESS_KEY := $(shell aws secretsmanager get-secret-value --secret-id=ci-user-aws-keys${randomSeed} | jq -r '.SecretString'| jq -r .secret_key)
-GITHUB_ORG := $(shell echo ${REPOSITORY} | cut -d "/" -f 2)
-GITHUB_REPO := $(shell echo ${REPOSITORY} | cut -d "/" -f 3)
-
 run:
 	@echo "Set CIRCLECI environment variables\n"
+	AWS_ACCESS_KEY_ID := $(shell aws secretsmanager get-secret-value --secret-id=ci-user-aws-keys${randomSeed} | jq -r '.SecretString'| jq -r .access_key_id)
+	AWS_SECRET_ACCESS_KEY := $(shell aws secretsmanager get-secret-value --secret-id=ci-user-aws-keys${randomSeed} | jq -r '.SecretString'| jq -r .secret_key)
+	GITHUB_ORG := $(shell echo ${REPOSITORY} | cut -d "/" -f 2)
+	GITHUB_REPO := $(shell echo ${REPOSITORY} | cut -d "/" -f 3)
```

Comment:
Moved these here so they aren't run for `summary`.  They are only needed by the `run` target.

Reply:
You will probably need org and repo to construct the path to the circleci pipeline.